hack/generate-operator-bundle.py

```diff
@@ -68,6 +69,13 @@
         }
     )
 
+# Copy all prometheus yaml files over to the bundle output dir:
+prom_files = [ f for f in os.listdir('deploy') if re.match(r'.*prometheus-k8s.*.yaml', f) ]
```

Comment:
nits:
- If you use `re.search`, you don't need the initial `.*` -- it doesn't use implicit anchors.
- The last `.` will theoretically match any character. If you were being precise, you should use `\.`.